### PR TITLE
trivial: make udisks partition proxy optional

### PR DESCRIPTION
**Pull request by superm1**
If a given block device doesn't advertise it, then don't check for it!.

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by superm1** [comment]
This appears to be too much, it's getting all sorts of partitions.  I'll need to pare it back further.

File: libfwupdplugin/fu-volume.c [hunk 1]
```diff
@@ -843,6 +868,7 @@ fu_volume_new_by_kind(const gchar *kind, GError **error)
 	devices = fu_common_get_block_devices(error);
 	if (devices == NULL)
 		return NULL;
+	g_info("Looking for volumes of type %s", kind);
```
Comment (hughsie) [inline libfwupdplugin/fu-volume.c:871]: g_debug here -- it's not something we want in syslog really

Comment (superm1) [reply libfwupdplugin/fu-volume.c:871]: Remember `g_debug` == `-vv` and `g_info` = `-v`

Comment (hughsie) [reply libfwupdplugin/fu-volume.c:871]: I'm thinking of g_log maybe. Too much relaxing on PTO. 🙂

**Review by hughsie — APPROVED** [review]

**Pull request merged**